src/CSVReader.java

```diff
@@ -1,61 +1,56 @@
 /**
  * Created by kenshoo-linux on 14/07/16.
  */
-import java.io.BufferedReader;
-import java.io.FileNotFoundException;
-import java.io.FileReader;
-import java.io.IOException;
+import java.io.*;
 
-public class ReadCSV {
+
+public class CSVReader implements Closeable{
 
     public static void main(String[] args) {
 
-        ReadCSV obj = new ReadCSV();
+        CSVReader obj = new CSVReader();
         obj.run();
 
     }
 
     public void run() {
 
-        String csvFile = "/home/kenshoo-linux/develop/projects/corejavaexercise (copy)/macycExampleFile.csv - macycExampleFile.csv.csv";
+        final String csvFile = "/home/kenshoo-linux/develop/projects/corejavaexercise (copy)/macycExampleFile.csv - macycExampleFile.csv.csv";
         BufferedReader br = null;
-        String line = "";
-        String str="";
+        /////////Nir asked to add "private static final" -why isn't it working?
+        //private static final String csvSplitBy = ",";
```

Comment:
No here. You add as a class field. The Java conversion for these are upper case so it will be CSV_SPLIT_BY
or (a name I would prefer): SEPARATOR (I think it is clear that it is used a the line separator since we have a single type of separator)